client/account-recovery/reset-password/reset-password-form/index.jsx

```diff
@@ -15,31 +15,46 @@ import FormFieldset from 'components/forms/form-fieldset';
 import FormLegend from 'components/forms/form-legend';
 import ResetOptionSet from './reset-option-set';
 
-import { getAccountRecoveryResetOptions } from 'state/selectors';
+import {
+	pickResetMethod,
+	requestReset,
+} from 'state/account-recovery/reset/actions';
+
+import {
+	getAccountRecoveryResetUserData,
+	getAccountRecoveryResetOptions,
+	getAccountRecoveryResetPickedMethod,
+	getAccountRecoveryResetRequestError,
+	isRequestingAccountRecoveryReset,
+} from 'state/selectors';
 
 export class ResetPasswordFormComponent extends Component {
 	static defaultProps = {
 		translate: identity,
+		pickedMethod: null,
+		userData: {},
+		requestError: null,
+		isRequesting: false,
```

Comment:
Are the defaults necessary if its connected to the Redux selectors which return defaults anyway? Particularly those that are `isRequired` in the proptypes - I'm thinking that if those properties aren't specified it might be better to see an error than silently return empty data.